### PR TITLE
Delete nested field

### PR DESCRIPTION
**Pull request by repeatedly**
Additional feature for `record_accessor`.
This is useful for several mutation plugins.

Ref #1578

### CODE REVIEW

**Comment by repeatedly** [comment]
@mururu Could you check this?

File: test/plugin/test_filter_record_transformer.rb [hunk 1]
```diff
@@ -83,6 +84,14 @@ def filter(config, msgs = [''])
       end
     end
 
+    test 'remove_keys with nested key' do
+      config = CONFIG + %[remove_keys $.nest.k1]
+      filtered = filter(config)
+      filtered.each_with_index do |(_t, r), i|
+        assert_not_include({'k2' => 'v2'}, r['nest'])
```
Comment (mururu) [inline test/plugin/test_filter_record_transformer.rb]: This line should be `assert_not_include(r['nest'], 'k1')`.

Comment (repeatedly) [reply test/plugin/test_filter_record_transformer.rb]: right. just fixed.

**Comment by mururu** [comment]
Looks good.

**Pull request merged**